.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,10 @@
 
 [Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
 
-- [ ] Title of the PR starts with chart name (e.g. [mychartname])
+- [ ] Title of the PR starts with type and scope, type must be one of feat, new, fix, perf, refactor, chore, docs (e.g. fix(node-analyzer):)
```

Comment:
Considering we want every PR/commit to have type & scope as a prefix in the Changelog maybe we can make it mandatory in the lint-test (pull-request based workflow) for every PR to have that. what do you think?
Also, this example can be made more descriptive like what would a fix for 2 charts agent and rapid-response look like. i am assuming fix(agent,rapid-response):

Reply:
The match is done via regex, this is an example https://regex101.com/r/WbpM78/1

As it stands `fix(agent,rapid-response):` would not work, we can play around with the regex to include something like that, anyway i'm not a huge fan of the extremely long and unreadable commit messages we have right now
My idea would be that if your change affects more than one chart the "scope" could be something like global, or all (e.g. fix(global): add automated changelogs)

Reply:
I agree on making it mandatory in the PR, we should be able to easily check that

Reply:
fyi improved the regex a bit, now `fix(agent,rapid-response):` is allowed, also made a few `README` adjustments